content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -51,7 +51,7 @@ <h3>Podsumowanie:</h3>
                 </div>
                 <div class="content__box content__box_fill">
                         <p><i>
-                            Pod to grupa jednego lub wielu kontenerów aplikacji (takich, jak Docker czy rkt) zawierający współdzielony storage (volumes), adres IP i informacje, jak mają być uruchamiane.
+                            Pod to grupa jednego lub wielu kontenerów aplikacji (takich, jak Docker czy rkt) zawierająch współdzieloną przestrzń dyskową (volumes), adres IP i informacje, jak mają być uruchamiane.
```

Comment:
```suggestion
                            Pod to grupa jednego lub wielu kontenerów aplikacji (takich, jak Docker czy rkt) zawierających współdzieloną przestrzeń dyskową (volumes), adres IP i informacje, jak mają być uruchamiane.
```

Reply:
Thanks for your contribution! Please create a new PR with these two typos.

Reply:
Sorry, for that, I was struggling with this PR and I have to fork repo a few times and I finally did typos...

Reply:
No worries!